.gitignore

```diff
@@ -63,7 +63,7 @@ instance/
 
 # Sphinx documentation
 docs/_build/
-docs/gen_modules
```

Comment:
why is this getting removed?

Reply:
This is now generated inside `docs/_staging/gen_modules`, and is included in that ignore.

Reply:
We might want to do it in two stage, first keep the old cleanup so it cleans up the files that is generated by the problem